### ISSUE TITLE
Crash while matching brackets

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#1447]
Leo Version: 6.1, OS: Windows 10 

When a extra bracket is present, and the command _match-brackets_ is issued several times. the following error may be raised:
```
TypeError: '<' not supported between instances of 'NoneType' and 'int'
```
I have gotten this to occur when I put the cursor just to the left of the right-hand most bracket (the one that is not matched) in the code snippet below:
```
x = [(1,2), (3,4)]]
```

**Comment by largo84** [comment]
I get the same error (same Leo version and Win10).

**Comment by edreamleo** [comment]
Fixed at 4c34367 in devel.
